client/a8c-for-agencies/components/items-dashboard/item-preview-pane/item-preview-pane-header/index.tsx

```diff
@@ -44,32 +46,38 @@ export default function ItemPreviewPaneHeader( {
 	return (
 		<div className={ classNames( 'item-preview__header', className ) }>
 			<div className="item-preview__header-content">
-				<SiteFavicon
-					blogId={ itemData.blogId }
-					fallback={ siteIconFallback }
-					color={ itemData.color }
-					className="item-preview__header-favicon"
-					size={ size }
-				/>
+				{ isSiteData && (
+					<SiteFavicon
+						blogId={ itemData.blogId }
+						fallback={ siteIconFallback }
+						color={ itemData.color }
+						className="item-preview__header-favicon"
+						size={ size }
+					/>
+				) }
```

Comment:
I know it still has some linked pieces for specific contexts, but if we can keep this component totally agnostic, better.

So instead, I'd introduce this property in the `ItemData` object: `withIcon` (by default true), and if it's false, we won't render the icon.